ReflectionAnalyzers.Tests/REFL046DefaultMemberMustExistTests/ValidCode.cs

```diff
@@ -0,0 +1,132 @@
+namespace ReflectionAnalyzers.Tests.REFL046DefaultMemberMustExistTests
+{
+    using System.Reflection;
+    using Gu.Roslyn.Asserts;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using NUnit.Framework;
+
+    [DefaultMember("Analyzer")]
```

Comment:
Is the attribute used for anything?

Reply:
It is not, not sure why I added that. Probably a copy/paste error.